### PR TITLE
remove Attributes from IstioEndpoint

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
cleanup

### CODE REVIEW

File: pilot/pkg/model/service.go [hunk 1]
```diff
@@ -73,6 +73,8 @@ type Service struct {
 	// Address specifies the service IPv4 address of the load balancer
 	Address string `json:"address,omitempty"`
 
+	// Protect concurrent ClusterVIPs read/write
+	Mutex sync.RWMutex
```
Comment (ramaraochavali) [inline pilot/pkg/model/service.go]: nit: add a new line below this line

**Comment by howardjohn** [comment]
Are these not used anywhere? I guess not if you could remove it

**Comment by hzxuzhonghu** [comment]
yes, it is safe to remove

**Pull request merged**